### ISSUE TITLE
Matrix and IRC

### ISSUE THREAD

**Issue by ShalokShalom** [go-gitea/website#62]
IRC and Matrix is available as well - reflect that in the homepage.

**Comment by strk** [comment]
Agreed. See also a previous attempt at this in #56 - that one _replaced_ discord with matrix, maybe *adding* matrix and IRC would have wider acceptance

**Comment by ShalokShalom** [comment]
For your information: Konversation 2.0 includes support for both, Matrix and IRC;

https://userbase.kde.org/Konversation/Konvi2x

**Comment by lunny** [comment]
Currently, discord is still official chat channels. Why changed to others?

**Comment by ShalokShalom** [comment]
Lets ask the user base. 

I see 692 users in the IRC room, nearly all of them marked with a (m) for Matrix. 
And now lets look into the Discord room: 52 users.

I think that speaks a clear language. 

In my humble opinion: 

The registration process in Discord is a pain, the whole interface is cluttered and filled with unnecessary stuff, plus the huge load of self-promotion that it puts in my way until I came from the simple 
"I like to ask one quick question" through an adventure of distraction into the actual room where not even a fractional part of the community sits, while that room is, in fact, the only promoted one. 

Plus, it's non-free. 

That this is even a question, wonders me. 

The goal of Gitea is to create an open source alternative to Github. 

So, to use an alternative to other non-free solutions seems legit. 

The community has already decided, does this not get accepted?

**Comment by lunny** [comment]
![image](https://user-images.githubusercontent.com/81045/34901649-e4ccba9e-f7d2-11e7-9bc0-0aff89d4e498.png)
Discord has 942 members but we only show online users.

**Comment by ShalokShalom** [comment]
Matrix shows online members or offline?

Many people who once logged in and never come back might count more as a negative argument.

**Comment by lafriks** [comment]
@ShalokShalom matrix shows all users, also offline. That's one of the problem of it that for room with so many users it's not even possible to see users that are online... (or could be that currently no users in matrix are online at all)

**Comment by ShalokShalom** [comment]
Fine, I see the point. 

None the less, I prefer the free choice.

**Comment by lafriks** [comment]
Of free choices I would prefer mattermost but we would need to host it ourself then

**Comment by ShalokShalom** [comment]
Well, I mean a free choice among those who are available. Currently, two are hidden and one is promoted.
These two are FOSS, the other one not. For a FOSS project, this seems somewhat obscure.

**Comment by lunny** [comment]
Maybe we should use mattermost to do this in future.

**Comment by strk** [comment]
> Maybe we should use mattermost to do this in future.

Fun comment, in an issue titled "Matrix and IRC"

**Comment by ShalokShalom** [comment]
Well, Matrix is a bit more decentralized, therefore is Mattermost really well written, from what I hear. Your choice, open another issue maybe?

**Comment by lafriks** [comment]
But matrix lacks some features like private rooms
